parsecmdline_pk.lua

```diff
@@ -56,6 +58,7 @@ local function getopts(opt_meta, arg, usage)
                 opts[opt] = true
             end
         else
+            proc = false
```

Comment:
This means that on the first non-option (i.e. anything not starting with a dash), the following arguments will also be collected into `args` (positional arguments). I'd rather have only `--` stopping the processing of options and remove this line, so that options and positional arguments can be specified in any order. (Currently, `mgrep.lua`'s documentation string states to specify `<typeName>::<memberName>` first, but in reality -- before this change -- it doesn't/didn't matter.)